lib/logcat.js

```diff
@@ -106,6 +114,17 @@ class Logcat {
   getAllLogs () {
     return this.logs;
   }
+
+  async clear () {
```

Comment:
does it make a sense to have a dedicated method in appium-adb for such purpose?

Reply:
This is just in the `Logcat` class, not the `appium-adb` exposed class.

Reply:
I meant whether it makes sense to create such external endpoint and expose it to the public instead of having it as a class method?